api/v1alpha1/spinappexecutor_types.go

```diff
@@ -61,7 +61,20 @@ type SpinAppExecutorStatus struct {
 
 // OtelConfig is the supported environment variables for OpenTelemetry
 type OtelConfig struct {
-	Endpoint string `json:"endpoint"`
+	// Sets OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+	OtelExporterOtlpEndpoint string `json:"otel_exporter_otlp_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_TRACES_ENDPOINT environment variable
+	OtelExporterOtlpTracesEndpoint string `json:"otel_exporter_otlp_traces_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_METRICS_ENDPOINT environment variable
+	OtelExporterOtlpMetricsEndpoint string `json:"otel_exporter_otlp_metrics_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_LOGS_ENDPOINT environment variable
+	OtelExporterOtlpLogsEndpoint string `json:"otel_exporter_otlp_logs_endpoint"`
+
+	// Sets OTEL_SERVICE_NAME environment variable
+	OtelServiceName string `json:"otel_service_name"`
```

Comment:
we probably shouldn't attach a ServiceName to the executor - I assumed we were gonna populate that with app names

Reply:
@calebschoepp had me add this. Maybe he can elaborate?

Reply:
We don't yet have the functionality to populate service name based off app names. I also don't think that feature is trivial b/c most open telemetry libraries are very good about allowing you to dynamically set the resource. This env var is a workaround for that reality.

@endocrimes if you still feel strongly that we don't have this I'm fine to just remove it. I want to get this PR in ASAP.

Reply:
I think we should think this through - so drop from this PR